File: 프로그래머스/unrated/140108. 문자열 나누기/문자열 나누기.py
def solution(s):
    answer = 0
    c1 = 0
    c2 = 0
    for i in s:
        if c1 == c2:
            answer += 1
            k = i
        if k == i:
            c1 += 1
        else:
            c2 += 1
    return answer

File: 프로그래머스/unrated/140108. 문자열 나누기/문자열 나누기.java
class Solution {
    public int solution(String s) {
        int answer = 0;
        char c = s.charAt(0);
        int count1 = 0;
        int count2 = 0;
        
        for (int i =0; i < s.length(); i++) {
            if(count1 == count2) {
                answer++;
                c = s.charAt(i);
            }
            
            if (c == s.charAt(i)) {
                count1++;
            }
            else {
                count2++;
            }
            
        }
        return answer;
    }
}